code/modules/client/preference_setup/loadout/gear_tweaks.dm

```diff
@@ -110,7 +110,10 @@ Content adjustment
 			continue
 		else
 			path = 	contents[metadata[i]]
-		new path(I)
+		if(path)
+			new path(I)
+		else
+			log_debug("Contents Tweak: Tried to spawn invalid [path] in [I]!")
```

Comment:
but-but. `path` is `null` here. It'll always say, `tried to spawn invalid null in [I]`.